config/schedule.rb

```diff
@@ -154,6 +154,10 @@
     trigger: health_trigger,
     interruptable: true,
   },
+  {
+    task: 'driver:hmis_external_apis:update_unit_availability',
+    frequency: 3.minutes,
```

Comment:
Ah oops! Sorry this is missing a trigger so that it's only enabled for the prod hmis instance. I just commented this job out for now to be on the safe side and added a note to enable it in our launch plan doc. Seem okay?

Reply:
We need this to be happening on day 1 of launch, it shouldn't require a new deploy. If there is no way to turn it on and off without doing a code deploy, we'll just have to leave it on. It's a no-op for environments that don't have (1) the HMIS enabled, and (2) an "MPER" credential present. Maybe we should add a comment stating that.